framework/src/modules/http_api/controllers/transactions.js

```diff
@@ -138,19 +138,24 @@ TransactionsController.postTransaction = function(context, next) {
 	return channel.invoke('chain:postTransaction', [
 		transaction,
 		(err, data) => {
-			if (err) {
-				if (err instanceof ApiError) {
-					context.statusCode = err.code;
-					delete err.code;
-				}
+			let error = null;
```

Comment:
I think @lsilvs was mentioning we shouldn't use `invoke` with callback

Reply:
Being addressed by @michielmulders on another PR.